osutil/udev/netlink/conn.go

```diff
@@ -47,6 +48,10 @@ func (c *UEventConn) Connect(mode Mode) (err error) {
 		syscall.Close(c.Fd)
 	}
 
+	// this must be rather large for some events, see the reference implementation from systemd:
+	// https://github.com/systemd/systemd/blob/0d92a3088a50212f16bf72672832b2b61dfca551/src/udev/udevadm-monitor.c#L73
```

Comment:
AFAICT the systemd code changes socket receive buffer size at the kernel level via `setsockopt(SOL_SOCKET, SO_RECVBUF)`. This isn't exactly the same as we do. I would like to learn more about the problem here. 

Also what udevadm does is interesting. Looks like the default buffer size on my machine is 52 pages:
```
In [13]: s = socket.socket(socket.AF_NETLINK, socket.SOCK_RAW)                         

In [14]: s.getsockopt(socket.SOL_SOCKET, socket.SO_RCVBUF)                             
Out[14]: 212992
```
Wonder why they'd force it to 128k rather than have at least 128k.

Reply:
Interesting, FWTW what we had so far was Getpagesize(), which on my system is 4096 bytes.

Reply:
It's actually 128MB, not 128kB, my bad.